packages/react-components/react-search/docs/Spec.md

```diff
@@ -2,23 +2,79 @@
 
 ## Background
 
-_Description and use cases of this component_
+`SearchBox` is a text input field abstracting `<input type="search" />`.
 
 ## Prior Art
 
-_Include background research done for this component_
+- [Ant Design](https://ant.design/components/input)
+- [Carbon](https://carbondesignsystem.com/components/search/usage/)
+- [Lightning Design](https://www.lightningdesignsystem.com/components/input/)
+- [Material UI](https://mui.com/material-ui/react-text-field/)
+- [Spectrum](https://react-spectrum.adobe.com/react-spectrum/SearchField.html)
 
-- _Link to Open UI research_
-- _Link to comparison of v7 and v0_
-- _Link to GitHub epic issue for the converged component_
+### Comparison of v8 and v0
+
+#### v8
+
+The v8 SearchBox component is implemented as a text input field that lacks a type attribute.
+
+The v8 component supports:
+
+- a customizable leading icon, with the default being a search icon
+- hiding the leading icon when the search box is in focus by default, to allow the search content to span the entire width of the search box
```

Comment:
Another point that is worth discussing is how to v8 and v0 behave in RTL. I assume everything is flipped but it would be good to document the behavior here.